modules/passkey/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
```

Comment:
Could be removed from [docker version 25.05](https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-optional)

Reply:
https://github.com/safe-global/safe-modules/pull/431